### ISSUE TITLE
Open Terminal error if the path contains space

### ISSUE THREAD

**Issue by Mazzupa** [files-community/files-uwp#1077]
**Describe the bug**
When I click on Open in Terminal if the path cotains space it would gives you an error.

**To Reproduce**
1. Go to a directory wich path contains a space
2. Click on Open in Terminal (Windows terminal)

**Example**
The directory I try to open is: C:\Users\Patrick Mazzucchetti\Downloads
After clicking on Open In Terminal I get: [errore 0x80070002 durante il lancio di `Mazzucchetti\Downloads']

**Expected behavior**
The terminal should be opend in the correct directory despite the space in the path.

**Desktop (please complete the following information):**
 - OS Version: Windows 10.0.19592 build 19592
 - App version: v0.9.2.0

**Comment by chuchanpr** [comment]
Issue reproducible on 0.10.2.0

**Comment by tsvietOK** [comment]
Can't reproduce on v0.10.2
![image](https://user-images.githubusercontent.com/20501502/84697300-94fedb80-af56-11ea-98bd-2c45a0bfbad2.png)

**Comment by xmha97** [comment]
```
{
  "version": 1,
  "defaultTerminalPath": "wt.exe",
  "terminals": [
    {
      "name": "Windows Terminal",
      "path": "wt.exe",
      "arguments": "new-tab -p PowerShell pwsh -workingdirectory \"{0}\"",
      "icon": ""
    }
  ]
}
```

**Comment by yaichenbaum** [comment]
@Mazzupa This issue should be resolved in v0.13.
